### ISSUE TITLE
Tiến độ tiến hành chương Multilayer Perceptrons

### ISSUE THREAD

**Issue by duythanhvn** [aivivn/d2l-vn#783]
Mình hy vọng chương **Multilayer Perceptrons** này chúng ta có thể hoàn thiện vào 15/04/2020.

## Giai đoạn 1 `62 parts`
- [x] 4. [Multilayer Perceptrons Index](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+index_vn+label%3A%22chapter%3Amultilayer-perceptrons%22) `01 part`
- [x] 4.1. [Multilayer Perceptrons](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+mlp_vn) `08 parts`
- [x] 4.2. [Implementation of Multilayer Perceptron from Scratch](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+mlp-scratch_vn) `03 parts`
- [x] 4.3. [Concise Implementation of Multilayer Perceptron](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+mlp-gluon_vn) `01 part`
- [x] 4.4. [Model Selection, Underfitting and Overfitting](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+underfit-overfit_vn) `09 parts`
- [x] 4.5. [Weight Decay](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+weight-decay_vn) `06 parts`
- [x] 4.6. [Dropout](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+dropout_vn) `06 parts`
- [x] 4.7. [Forward Propagation, Backward Propagation, and Computational Graphs](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+backprop_vn) `05 parts`
- [x] 4.8. [Numerical Stability and Initialization](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+numerical-stability-and-init_vn) `05 parts`
- [x] 4.9. [Considering the Environment](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+environment_vn) `10 parts`
- [x] 4.10. [Predicting House Prices on Kaggle](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+kaggle-house-price_vn) `08 parts`

## Giai đoạn 2 `38 parts`
- [x] 4. [Multilayer Perceptrons Index](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+index_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3Amultilayer-perceptrons%22) `01 part`
- [x] 4.1. [Multilayer Perceptrons](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+mlp_vn+label%3A%22status%3A+phase+2%22) `04 parts`
- [x] 4.2. [Implementation of Multilayer Perceptron from Scratch](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+mlp-scratch_vn+label%3A%22status%3A+phase+2%22) `02 parts`
- [x] 4.3. [Concise Implementation of Multilayer Perceptron](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+mlp-gluon_vn+label%3A%22status%3A+phase+2%22) `01 part`
- [x] 4.4. [Model Selection, Underfitting and Overfitting](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+underfit-overfit_vn+label%3A%22status%3A+phase+2%22) `05 parts`
- [x] 4.5. [Weight Decay](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+weight-decay_vn+label%3A%22status%3A+phase+2%22) `03 parts`
- [x] 4.6. [Dropout](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+dropout_vn+label%3A%22status%3A+phase+2%22) `05 parts`
- [x] 4.7. [Forward Propagation, Backward Propagation, and Computational Graphs](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+backprop_vn+label%3A%22status%3A+phase+2%22) `03 parts`
- [x] 4.8. [Numerical Stability and Initialization](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+numerical-stability-and-init_vn+label%3A%22status%3A+phase+2%22) `03 parts`
- [x] 4.9. [Considering the Environment](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+environment_vn+label%3A%22status%3A+phase+2%22) `06 parts`
- [x] 4.10. [Predicting House Prices on Kaggle](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+kaggle-house-price_vn+label%3A%22status%3A+phase+2%22) `05 parts`

**Comment by tiepvupsu** [comment]
Tạm thời chưa lên issue chương này vội nhé em. Lên revise chương 3 trước. Nếu chưa có file nào của chương 3 xong giai đoạn 1 thì mình tập trung push các issue/PR đó.

Dàn trải quá sẽ khó quản lý được chất lượng.

**Comment by duythanhvn** [comment]
<details open>
  <summary> 4.1. Multilayer Perceptrons - Phase 1 (In Progress)</summary>

- [ ] Phần 1: di-mi-ta
- [ ] Phần 2: thunguyenuehk39
- [ ] Phần 3: nguyenduydu
- [ ] Phần 4: 
- [ ] Phần 5: 
- [ ] Phần 6: 
- [ ] Phần 7: 
- [ ] Phần 8: 
</details>
<details>
  <summary> 4.1. Multilayer Perceptrons - Phase 2 (Pending)</summary>


</details>

**Comment by tiepvupsu** [comment]
Đặt mục tiêu 15/4 nhé @duythanhvn.
